util/fastCross3.m

```diff
@@ -0,0 +1,7 @@
+function X = fastCross3(U, V)
```

Comment:
excellent (although I wish we didn't have to do this sort of thing!).  why the 3 at the end of the name?